subworkflows/local/depths.nf

```diff
@@ -53,9 +61,10 @@ workflow DEPTHS {
         .collectFile(name:'sample_groups.tsv'){ meta, reads -> meta.id + '\t' + meta.group + '\n' }
 
     // Filter MAG depth files: use only those for plotting that contain depths for > 2 samples
+    // as well as > 2 bins
     ch_mag_depths_plot = MAG_DEPTHS.out.depths
         .map { meta, bin_depths_file ->
-            if (getColNo(bin_depths_file) > 2) [ meta, bin_depths_file ]
+            if (getColNo(bin_depths_file) > 2 && getRowNo(bin_depths_file) > 2) [ meta, bin_depths_file ]
```

Comment:
Does the file have a header, and if so should this then be `> 3`?

Reply:
it does have a header, yes. but its actually ok to have only 2 bins... or at least, mag doesnt fail for that. this check is meant to catch the case of a single bin.